ApexOSA/polarimeter.py

```
@@ -0,0 +1,226 @@
+from .Common import Send, Receive
+import sys
+from random import random, randint
+from math import sqrt
+from .Errors import *
```

Ruff (0.12.2)

`from .Errors import *` used; unable to detect undefined names

(F403)